src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__picture {
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__product-title {
+    font-size: $mainElementsSize;
+    font-weight: 500;
+    line-height: $mainLineHeight;
+
+    @include block-distance;
```

Comment:
Reset standart h1 styles and then change margin-block to margin-bottom. There trouble can be in tests and they are not using margin-block in logic, as a variant. So try it